src/commands/diff/README.md

```diff
@@ -1,14 +1,33 @@
 # Diff
-The `diff` command compares `content models` between two environments and prints the difference in the format of [Kontent.ai Management API](https://kontent.ai/learn/docs/apis/openapi/management-api-v2/) operations. You can compare two environments by providing parameters for source and target environments (environment ID and MAPI key), or compare the target environment with a folder model created by [generate-sync-model](#generate-sync-model) command.
+The `diff` command compares **content models** of two environments and outputs the difference either to a command line interface or to a standalone HTML file for improved visual representation and sharing purposes. Two environments can be compared to each other directly by providing parameters for both source and target (environment ID and MAPI key). Alternatively, you can also compare the target environment with a folder model created by [generate-sync-model](#generate-sync-model) command.
+
+## Default output
+
+By default, `diff` command outputs to CLI, in a slightly altered format of [Kontent.ai Management API](https://kontent.ai/learn/docs/apis/openapi/management-api-v2/) operations for each modified entity. This can be suitable for smaller diffs or if you want to sync changes manually by running individual operations.
```

Comment:
`if you want to sync changes manually by running individual operations.` I am not sure if you understand this :/ we don't have partial sync yet

Reply:
changed to `if you want to inspect individual operations`